frontend-react/src/components/Admin/CompareJsonModal.tsx

```diff
@@ -73,14 +83,23 @@ export const ConfirmSaveSettingModal = forwardRef(
                 getEditedText: (): string => {
                     return diffEditorRef?.current?.getEditedText() || newjson;
                 },
+                getOriginalText: (): string => {
+                    return diffEditorRef?.current?.getOriginalText() || newjson;
```

Comment:
ah good catch, thanks! I actually wound up NOT needing this at all, but I left it in because I thought it would be useful to a future dev. Instead, I introduced a future confusing bug! Yay! I'll submit a little PR to correct this.